template/react-redux-application/src/actions/index.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+import * as types from '../constants/ActionTypes';
+
+export const updateName = (name: string): Object => {
```

Comment:
Here is an example.

https://flowtype.org/try/#0PQKgBAAgZgNg9gdzCYAoVAXAngBwKZgZwAmcAggMYYCWcAdmALxgDehueAXGAORkAi-APoAVAPL8xPADRgAbgEMYAVy5gAzhgBO1OgHMwAXzAAfVu3zceAJQCiAWTEA1W6IlTZ1YtzrKAtgBGeFoA3EYh6BT0mmBaeMTKFATMABQKVLR03ESklDT0AJRMAHysqGAaCNQYFAAWaRn0AHTY+EUs5RVgFArqBHyCbpI8nJ1dYMDAYAASwQQ9DOlJ6upg9DBYYAAGTYoqeDtjXXEYylqLjXS7SqoR4z19vHaOLkNSo+MVkzNz3QoXy1W602Oy8h0+sTwp3OYHS+SuXjuXWIeCgCmUMAwH0+3wAklAwFg4MowFA4Fo9FCNHA-AQ4ZkUuoirpCLVqKt1FUarUNBgFBg8LS6BhZAAxeBIPwKADWeFWwS05MZBSaRy+UxEWk26hpULZ+gAhGqwA14dxBThsAUkYZUIYIkA